dev-resources/template/marketplace/3_lrs.yml

```diff
@@ -0,0 +1,937 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "SQL LRS and DB Init Script"
+Parameters:
+  VPCStackName:
+    Description: Name of our VPC Stack
+    Type: String
+  # DB Details
+  DBStackName:
+    Description: DB Stack Reference (leave empty if ephemeral)
+    Type: String
+    Default: ''
+  SecretStackName:
+    Description: SSM Stack Reference
+    Type: String
+  # Lambda DB Init Script
+
+  # Server(s)
+  InstanceType:
+    Type: String
+    Description: EC2 Instance Type to launch.
+    Default: c5.large
+    AllowedValues:
+      - c5.large
+      - c5.xlarge
+      - c5.2xlarge
+      - c5.4xlarge
+      - c5.9xlarge
+      - c5.12xlarge
+      - c5.18xlarge
+      - c5.24xlarge
+  InstanceKeyName:
+    Description: Name of an EC2 key pair that can be used to connect to the server (optional)
+    Type: String
+  InstanceSSHCidr:
+    Description: CIDR Range for SSH Access to instances (Typically VPC CIDR Range)
+    Type: String
+    Default: "172.147.0.0/16"
+  InstanceHttpPort:
+    Description: Port on which the server runs HTTP
+    Type: String
+    Default: 8080
+  InstanceCreationPolicyResourceSignalTimeout:
+    Description: How long to wait for a resource signal on the deployment of an instance.  
+    Type: String
+    Default: PT15M
+  DefaultAdminUser:
+    Default: lrsqladmin
+    Type: String
+    Description: Username of seed lrs admin
+  DefaultAdminPass:
+    Description: Initial seed password for lrs admin
+    NoEcho: true
+    Type: String
+  LrsVersion:
+    Description: Version of SQL LRS to pull from Mapping of AMIs
+    Type: String
+    AllowedValues:
+      - v0713
```

Comment:
woof, this is the best they can do with semver? I can see this being an issue down the line...

Reply:
well the results of the amibuilder are just ami IDs, what im doing here is mapping from semver->region->ami ID. Unfortunately this does mean this template requires manual updates of those mappings for every release.